database_benchmarking_suite/README.md

```diff
@@ -0,0 +1,57 @@
+# Overview
+
+This project is aimed at benchmarking databases for the EVA/EGA/AMP accessioning services.
+
+# Pre-requisites
+
+Running the benchmark from a host machine requires a local installation of [Apache JMeter](https://jmeter.apache.org/). The full path to Apache JMeter is specified with the command-line switch **-j** in the benchmarking program. Please read the section below for more details.   
```

Comment:
I wouldn't introduce the `-j` argument just yet: it is not exactly applicable to the pre-requisites, and it explained below anyway.